image-overview-templates/new-names/security-proxy-auth-arm64.md

```diff
@@ -0,0 +1,19 @@
+$$./common/warning.md
+
+$$./common/quick-reference.md
+
+# Supported tags and respective Dockerfile links
+
+$$./common/version-minnesota.md |
+        - [Dockerfile](https://github.com/edgexfoundry/edgex-go/blob/v3.0.0/cmd/security-proxy-auth/Dockerfile)
+
+$$./common/quick-reference-cont-edgex-go.md
+
+$$./common/what-is-edgex.md
+
+# What's in this image?
+
```

Comment:
add `**arn64** version of the ` to @bnevis-i  description for the non-arm version

Reply:
**arm64**